frontend/app/components/toaster.tsx

```diff
@@ -12,9 +15,67 @@ export interface ToasterProps {
 export function Toaster({ toast }: ToasterProps) {
   useEffect(() => {
     if (toast?.message) {
-      enqueueSnackbar(<SnackbarMessage message={toast.message} />, { variant: toast.type });
+      enqueueSnackbar(toast.message, { variant: toast.type });
     }
   }, [toast]);
 
-  return <SnackbarProvider className="bg-green-700" anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }} />;
+  return (
+    <SnackbarProvider
+      anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
+      Components={{
+        default: DefaultSnackbar,
+        error: ErrorSnackbar,
+        info: InfoSnackbar,
+        success: SuccessSnackbar,
+        warning: WarningSnackbar,
+      }}
+      iconVariant={{
+        error: <XCircleIcon className="me-2 h-5 w-5" />,
+        info: <AlertCircleIcon className="me-2 h-5 w-5" />,
+        success: <CheckCircle2Icon className="me-2 h-5 w-5" />,
+        warning: <AlertTriangleIcon className="me-2 h-5 w-5" />,
+      }}
+    />
+  );
 }
+
+type SnackbarProps = ComponentProps<typeof MaterialDesignContent>;
```

Comment:
Use interface as recommended by Microsoft. You want to use interface with object type.

```suggestion
interface SnackbarProps extends ComponentProps<typeof MaterialDesignContent> { }
```